src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/PreferIsEmptyOverAnyTests.cs

```diff
@@ -0,0 +1,386 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Performance.PreferLengthCountIsEmptyOverAnyAnalyzer,
+    Microsoft.NetCore.CSharp.Analyzers.Performance.CSharpPreferLengthCountIsEmptyOverAnyFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Performance.PreferLengthCountIsEmptyOverAnyAnalyzer,
+    Microsoft.NetCore.VisualBasic.Analyzers.Performance.BasicPreferLengthCountIsEmptyOverAnyFixer>;
+
+namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
+{
+    public class PreferIsEmptyOverAnyTests
+    {
+        private static readonly DiagnosticResult ExpectedDiagnostic = new DiagnosticResult(PreferLengthCountIsEmptyOverAnyAnalyzer.IsEmptyDescriptor).WithLocation(0);
+
+        [Fact]
+        public Task TestLocalDeclarationAsync()
+        {
+            const string code = @"
+using System.Collections.Immutable;
+using System.Linq;
+
+public class Tests {
+    public void M() {
+        var array = ImmutableArray<int>.Empty;
+        _ = {|#0:array.Any()|};
```

Comment:
Please add a test where `array.Any()` is passed as an argument to a method that accepts a boolean. If the codefix crashes, you'll need to pass `getInnermostNodeForTie: true` to `FindNode`